src/connectors/youtube.js

```diff
@@ -140,7 +140,16 @@ function setupEventListener() {
 }
 
 function areChaptersAvailable() {
-	return Util.getTextFromSelectors(chapterNameSelector);
+	const text = Util.getTextFromSelectors(chapterNameSelector);
+
+	// SponsorBlock extension hijacks chapter element. Ignore it.
+	const sponsorBlockButton = document.getElementById('startSegmentButton');
+	if (sponsorBlockButton && (text === 'Non-Music' || text === 'Sponsor' || text === 'Credits/Endcards' || text === 'Intermission')) {
```

Comment:
```suggestion
	if (sponsorBlockButton && (text === 'Non-Music' || text === 'Sponsor' || text === 'Endcards/Credits' || text === 'Intermission')) {
```

Should it be this way round?


See: https://github.com/ajayyy/SponsorBlock/blob/1a0dd19cf0feff639e2e7943736409278c5fbee6/public/_locales/en/messages.json#L671

Reply:
Yep, my bad